tests/neg-macros/quote-type-variable-no-inference-2.check

```diff
@@ -0,0 +1,13 @@
+-- Warning: tests/neg-macros/quote-type-variable-no-inference-2.scala:5:22 ---------------------------------------------
+5 |    case '{ $_ : F[t, t]; () } => // warn // error
+  |                      ^
+  |                      Ignored bound <: Double
+  |
+  |                      Consider defining bounds explicitly:
+  |                        '{ type t <: Int & Double; ... }
```

Comment:
I wonder if we could take advantage of https://github.com/lampepfl/dotty/pull/17561 to automatically propose a rewrite that does that.

Reply:
I am considering doing that. The complication is finding the position where this needs to be inserted. It would be also be a bit more challenging if we consider handling two of these errors at the same time. I plan to try this in the near future.